assets/js/vue-apps/form-components/budget-input.test.js

```diff
@@ -62,14 +62,12 @@ describe('BudgetInput', () => {
         addItem('My thing 9', 6000);
         checkItemCount(10);
         checkTotal('£10,500');
-        checkWarning(
-            `Costs you would like us to fund must be less than £${maxBudget.toLocaleString()}.`
-        );
+        checkWarning(`£${maxBudget.toLocaleString()}.`);
```

Comment:
This fix (and below) is really dumb but I've spent almost half a day on it and fed up wasting time now. You can see the [original error here](https://travis-ci.org/biglotteryfund/blf-alpha/builds/564944792#L498) – no matter how I tried to fix this (passing in the `i18n` copy object from memory, passing it in explicitly with the values hard coded) it broke the component. Random things like the total would fail to compute. I couldn't work out what was causing this given the lengths I've gone to here to make the `copy` optional so I've resorted to simply checking that the error contains the number in question.

Reply:
Going forward we could do with a better pattern both for adding translations to client-side components and for testing them. I didn't want to use the `localise({ en: '...', cy: '...'})` pattern here because these translations are shared between the server/client.

Reply:
Yeah I can see this being a pain, personally for any vue components I like the pattern of a single property per string value (so translations happen outside the component) where we are more able to handle the variation. Although trying to use the localise pattern would be OK too, I think it's possibly overkill to try and share translations between server and client, so I'd be OK duplicating them.